pkgs/development/libraries/kde-frameworks/kde-wrapper.nix

```diff
@@ -61,13 +61,9 @@ stdenv.mkDerivation {
         fi
     done
 
-    ln -s "$env/share" "$out"
```

Comment:
That's OK! `buildEnv` is smart enough to avoid collisions if the same package ends up in `propagatedUserEnvPkgs` many times. What it is *not* smart enough to do is avoid collisions if you have two different packages that provide symlinks to the same destination! That's the root problem here.